Reneiski_Homework1/ReneiskiSelenium11/Program.cs

```diff
@@ -0,0 +1,239 @@
+﻿using NUnit.Framework;
```

Comment:
Please install Trailing Whitespace extensions and use it across the file
many whitespaces :( 
![image](https://user-images.githubusercontent.com/55654931/209944020-42fa625e-91db-4d0f-a6b0-768a3966a5b5.png)

Reply:
Done and fixed!